Tests/Unit/Pipeline/PipelineTest.php

```diff
@@ -17,12 +17,31 @@
 use ONGR\ConnectionsBundle\Pipeline\PipelineFactory;
 use Symfony\Component\EventDispatcher\EventDispatcher;
 
-/**
- * PipelineTest class.
- */
 class PipelineTest extends \PHPUnit_Framework_TestCase
 {
     /**
+     * Pipeline data provider.
+     *
+     * @return array
+     */
+    public function pipelineData()
+    {
+        // Modifier skips items which are string 'skip', anything else gets consumed.
```

Comment:
Modifier skips items which are equal to 'skip', anything else gets consumed.

sounds a bit better IMHO.